### PR TITLE
Adding class for required input to easily put a * in form-group with css

### PR DESCRIPTION
**Pull request by jnoelEFL**
Hi,

I've adding a class for required input. It's to easily put sign (* or another) in a form-group with css.

### CODE REVIEW

File: src/backform.js [hunk 1]
```diff
@@ -323,6 +323,12 @@
       if (!data.visible)
         this.$el.addClass(Backform.hiddenClassname);
 
+      // Adding class for required input to easily put a * in form-group with css
+      this.$el.removeClass(Backform.requiredClassname);
```
Comment (martindrapeau) [inline src/backform.js]: Many points:
1) Should be called requiredInputClassName.
2) You never define this property. Add it to the Backform namespace definition on line 31 as undefined.
3) Do not call removeClass if the property does not exist. So move that into the if block.
4) Remove the comment in the code. You may want to add a small section in the docs to showcase this property and its use case.

I also noticed that property hiddenClassname does not follow the camel-case convention. Should be hiddenClassName instead. May want to make that change as well.

**Comment by jnoelEFL** [comment]
Hi,

I correct my code

**Comment by martindrapeau** [comment]
Just edit the index.html and add a section below the existing "Visible Attribute as a function in Field" section.

**Comment by martindrapeau** [comment]
I merged your change but if you can take the time to add some docs, would be great.

**Pull request merged**